test/widgets/autocomplete_test.dart

```diff
@@ -48,18 +47,14 @@ Future<Finder> setupToComposeInput(WidgetTester tester, {
 
   prepareBoringImageHttpClient();
 
-  await tester.pumpWidget(
-    MaterialApp(
-      localizationsDelegates: ZulipLocalizations.localizationsDelegates,
-      supportedLocales: ZulipLocalizations.supportedLocales,
-      home: GlobalStoreWidget(
-        child: PerAccountStoreWidget(
-          accountId: eg.selfAccount.id,
-          child: MessageListPage(
-            narrow: DmNarrow(
-              allRecipientIds: [eg.selfUser.userId, eg.otherUser.userId],
-              selfUserId: eg.selfUser.userId,
-            ))))));
+  await tester.pumpWidget(const ZulipApp());
+  await tester.pump();
+  final navigator = await ZulipApp.navigator;
+  navigator.push(MessageListPage.buildRoute(accountId: eg.selfAccount.id,
+    narrow: DmNarrow(
+      allRecipientIds: [eg.selfUser.userId, eg.otherUser.userId],
+      selfUserId: eg.selfUser.userId,
+    )));
 
   // global store, per-account store, and message list get loaded
   await tester.pumpAndSettle();
```

Comment:
This comment is no longer accurate. Perhaps just cut it:
```suggestion
    )));
  await tester.pumpAndSettle();
```

And perhaps it can be reduced to just a `pump`?